lib/registries/addon/components/registries-overview-menu/component.ts

```diff
@@ -0,0 +1,30 @@
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import styles from './styles';
+import layout from './template';
+
+enum RegistrationState {
+    PENDING = 'pending',
+    EMBARGOED = 'embargoed',
+    PUBLIC = 'public',
+    WITHDRAWN = 'withdrawn',
+}
+
+export default class RegistriesOverviewMenu extends Component {
+    layout = layout;
+    styles = styles;
+
+    scientistName?: string = 'Curie';
```

Comment:
[Theres a utility for this!](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/app/utils/random-scientist.ts)